app/components/steps_component.rb

```diff
@@ -112,14 +101,25 @@ def back_link
       link_to(t('global.button.back'), '#', class: 'link cancel', data: data)
     end
 
+    # Lazy way of handling optional and required attributes
+    # - yield markup to with_model(f.object) and use c.attr? and c.required?
+    def with_model(model)
+      @model = model
+      yield
+    end
+
+    def attr?(attr)
+      @model.attrs.include?(attr)
+    end
+
+    def required?(attr)
+      @model.required_attrs.include?(attr)
```

Comment:
Ich würd noch raisen mit einer Erklärung, dass das ganze in einen with_model block gewrappt werden muss